components/bpmn/bpmn-explorer-web/src/web/config/config.json

```diff
@@ -1,17 +1,19 @@
 {
-    "app": {
-	"title" : "BPMN Explorer",
-	"author": "WSO2 Business Process Server",
-	"footer": "&copy; 2005 - 2015 WSO2 Inc. All Rights Reserved."
-    },
-    "server": {
-        "https": "%https.host%",
-        "http": "%http.host%",
-        "bpsHost":"",
-        "bpsPort":"",
-        "bpsTenantDomain": "",
-        "bpsTenantId" : ""
-    }
+  "app": {
+    "title": "BPMN Explorer",
+    "author": "WSO2 Business Process Server",
+    "footer": "&copy; 2005 - 2017 WSO2 Inc. All Rights Reserved.",
```

Comment:
Can we make footer year to  2018 in both config.json ?

Reply:
I did not change it since it's not relevant to this fix